cli/src/mdastToRst.ts

```diff
@@ -263,7 +269,7 @@ const visitors: {
         anchorName !== undefined,
         `unexpected refLink without anchorName from url: ${url}`
       );
-      c.add(` <${anchorName}>\``);
+      c.add(` <${anchorName}>\` `);
```

Comment:
ref trailing space, since non-whitespace immediately after a ref breaks the link and renders awfully.